### ISSUE TITLE
Fixup new LaTeX warnings

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#9645]
Issue overview
--------------

```shell
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1417::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1419::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1527::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1423::Label `field-chiller205-representation-file-name' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1427::Label `field-chiller205-representation-file-name' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1417::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1419::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1527::Label `' multiply defined.]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\watt ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\per ->\ERROR]
[LATEXERROR::doc/input-output-reference/src/overview/group-evaporative-coolers.tex:None::Undefined control sequence.\volumeFlowRate ->\ERROR]
```

```shell
[100%][28/28] Running utility command for zPDF_EngineeringReference
[LATEXWARNING::doc/engineering-reference/src/surface-heat-balance-manager-processes/outside-surface-heat-balance.tex:241::Label `table:atmospheric-variables-at-two-different' multiply defined.]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:1671::Missing $ inserted.<inserted text> $ l.1671 ...tails from: https://www.ahrinet.org/App_ Content/ahri/files/STANDAR... I've inserted a begin-math/end-math symbol since I think you left one out. Proceed, with fingers crossed.]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:1673::Missing $ inserted.<inserted text> $ l.1673]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:2307::Missing $ inserted.<inserted text> $ l.2307 ...tails from: https://www.ahrinet.org/App_ Content/ahri/files/STANDAR... I've inserted a begin-math/end-math symbol since I think you left one out. Proceed, with fingers crossed.]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:2309::Missing $ inserted.<inserted text> $ l.2309]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:3783::Missing $ inserted.<inserted text> $ l.3783 ...tails from: https://www.ahrinet.org/App_ Content/ahri/files/STANDAR... I've inserted a begin-math/end-math symbol since I think you left one out. Proceed, with fingers crossed.]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:3785::Missing $ inserted.<inserted text> $ l.3785]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:4366::Missing $ inserted.<inserted text> $ l.4366 ...tails from: https://www.ahrinet.org/App_ Content/ahri/files/STANDAR... I've inserted a begin-math/end-math symbol since I think you left one out. Proceed, with fingers crossed.]
[LATEXERROR::doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:4368::Missing $ inserted.<inserted text> $ l.4368]
```

ref https://github.com/NREL/EnergyPlus/commit/e83ea12cc356a58be689fa5f3159cd6252fec6f4 for the empty labels for eg.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version): All
 - Version of EnergyPlus (if using an intermediate build, include SHA): 22.2.0-IOFreeze, at 4872b63adc2630585108b0c9d0c5ebc492ca29e3

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jcyuan2020** [comment]
FYI. The first few lines have been fixed in #9629
```
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1417::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1419::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1527::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1423::Label `field-chiller205-representation-file-name' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1427::Label `field-chiller205-representation-file-name' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1417::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1419::Label `' multiply defined.]
[LATEXWARNING::doc/input-output-reference/src/overview/group-plant-equipment.tex:1527::Label `' multiply defined.]
```

**Comment by jmarrec** [comment]
Seems that develop no longer shows these warnings.
